common/src/main/java/com/microsoft/identity/common/exception/ErrorStrings.java

```diff
@@ -397,8 +397,18 @@ private ErrorStrings() {
     public static final String REGISTERED_SHARED_DEVICE_DELETED_ON_SERVER_ERROR_CODE =
             "registered_shared_device_deleted_on_server";
     public static final String REGISTERED_SHARED_DEVICE_DELETED_ON_SERVER_ERROR_MESSAGE =
-            "This device configuration was changed on the server. " +
-                    "Please return the device to your administrator to restart the registration of this device.";
+            "This device was deleted from the tenant. " +
+                    "Please return the device to your administrator to re-register this device.";
+
+    /**
+     * The device is registered with precreate/userless WPJ, and its registration was deleted by the admin.
+     * This is an irrecoverable error, and the admin has to re-prep the device.
+     * */
+    public static final String USERLESS_DEVICE_DELETED_ON_SERVER_ERROR_CODE =
+            "userless_device_deleted_on_server";
+    public static final String USERLESS_DEVICE_DELETED_ON_SERVER_ERROR_MESSAGE =
+            "This device was deleted from the tenant. " +
+                    "Please return the device to your administrator to re-register this device.";
```

Comment:
Looks to be the same value as used for the `REGISTERED_SHARED_DEVICE_DELETED_ON_SERVER_ERROR_MESSAGE` declared on line 399. Perhaps there could be a private constant called `DEVICE_DELETED_ON_SERVER_ERROR_MESSAGE` and both the error codes for userless and shared device reference that constant?